libnavigation-core/src/main/java/com/mapbox/navigation/core/RoutesProgressDataProvider.kt

```diff
@@ -0,0 +1,43 @@
+package com.mapbox.navigation.core
+
+import com.mapbox.navigation.base.route.NavigationRoute
+import com.mapbox.navigation.core.routealternatives.AlternativeMetadataProvider
+import com.mapbox.navigation.core.routealternatives.AlternativeRouteProgressDataProvider
+
+internal data class RoutesProgressData(
+    val primaryRoute: NavigationRoute,
+    val primaryRouteProgressData: RouteProgressData,
+    val alternativeRoutesProgressData: List<Pair<NavigationRoute, RouteProgressData?>>
```

Comment:
Not important suggestion, just a matter of taste 
```suggestion
    val alternativeRoutesProgressData: Map<NavigationRoute, RouteProgressData>
```

Reply:
I want to keep the order.